src/components/LoginRegister/LoginInput.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { useApolloClient } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+import { Input, Link, Button, Heading, Flex, VStack } from '@chakra-ui/react';
+import { login } from '../../data/actions/auth';
+
+const LoginInput = () => {
+  const client = useApolloClient();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [submitState, setSubmitState] = useState('');
+  const dispatch = useDispatch();
+
+  const handleClick = async e => {
+    e.preventDefault();
+    dispatch(login(email, password, client)).then(
+      loggedIn => {
+        if (!loggedIn) setSubmitState('FAIL');
+      },
+      error => {
+        setSubmitState('FAIL');
+      },
+    );
+  };
+
+  const onChangeEmail = e => {
+    setEmail(e.target.value);
+  };
+
+  const onChangePass = e => {
+    setPassword(e.target.value);
+  };
+
+  return (
+    <form onSubmit={e => handleClick(e)}>
+      <VStack spacing="24px">
+        <Flex w="100%" justify="left">
+          <Heading as="h1">Welcome</Heading>
+        </Flex>
+        <Input type="email" name="email" placeholder="Email" value={email} onChange={onChangeEmail} isRequired={true} />
+        <Input
+          type="password"
+          name="password"
+          placeholder="Password"
+          value={password}
+          onChange={onChangePass}
+          mb="8px"
+          isRequired={true}
+        />
+      </VStack>
+      <Link>Forgot Password?</Link>
+      {submitState === 'FAIL' && (
+        <Flex w="100%" justify="left">
+          <p>Oops! Check your credentials and try again.</p>
+        </Flex>
+      )}
+      <Flex justify="center" mt="24px">
+        <Button role="link" width="131px" height="43px" type="submit">
+          Sign In
+        </Button>
+      </Flex>
```

Comment:
ok so I just played around w/ Chakra and styling a bit more 

* Move `VStack` to wrap everything (just like in `RegisterInput`)
* Use `alignSelf` rather than creating a new `Flex` component (because `VStack itself creates a flex with align-items: center`
* For boolean props, you can just pass the prop name to set it to true, so `isRequired` rather than `isRequired={true}`

```suggestion
      <VStack spacing="24px">
        <Heading alignSelf="flex-start" size="h1" as="h1">
          Welcome
        </Heading>
        <Input type="email" name="email" placeholder="Email" value={email} onChange={onChangeEmail} isRequired />
        <Input
          type="password"
          name="password"
          placeholder="Password"
          value={password}
          onChange={onChangePass}
          mb="8px"
          isRequired
        />
        <Link alignSelf="flex-start">Forgot Password?</Link>
        {submitState === 'FAIL' && <Text alignSelf="flex-start">Oops! Check your credentials and try again.</Text>}
        <Button role="link" width="131px" height="43px" type="submit">
          Sign In
        </Button>
      </VStack>
```

end result is it still looks the same lool

![image](https://user-images.githubusercontent.com/26701004/109670256-ed0b5b80-7b40-11eb-869e-996d60f5c43d.png)

Reply:
Fixed, thanks!